homeassistant/components/philips_js/media_player.py

```diff
@@ -80,8 +80,7 @@ def __init__(
     ) -> None:
         """Initialize the Philips TV."""
         self._tv = coordinator.api
-        self._sources = {}
-        self._channels = {}
```

Comment:
`self._channels` was never used, so I figured it must be a leftover from a previous refactor.